kernel/build/config-amd64

```diff
@@ -3795,16 +3797,22 @@ CONFIG_FB_TILEBLITTING=y
 # CONFIG_FB_ARC is not set
 # CONFIG_FB_ASILIANT is not set
 # CONFIG_FB_IMSTT is not set
-# CONFIG_FB_VGA16 is not set
+CONFIG_FB_VGA16=y
 # CONFIG_FB_UVESA is not set
-# CONFIG_FB_VESA is not set
+CONFIG_FB_VESA=y
 CONFIG_FB_EFI=y
 # CONFIG_FB_N411 is not set
 # CONFIG_FB_HGA is not set
 # CONFIG_FB_OPENCORES is not set
 # CONFIG_FB_S1D13XXX is not set
-# CONFIG_FB_NVIDIA is not set
-# CONFIG_FB_RIVA is not set
+CONFIG_FB_NVIDIA=y
```

Comment:
i guess this gets enabled by enabled `CONFIG_FB_DDC`?

Reply:
no, I explicitly enabled it actually.